test/JDBC/expected/db_collation/ISC-Check-Constraints-vu-verify.out

```diff
@@ -74,7 +74,7 @@ isc_check_constraints_db1#!#dbo#!#test_tsql_const_c_varbinary_check#!#((c_varbin
 isc_check_constraints_db1#!#dbo#!#test_udd_c_udd_check#!#((c_udd < 133.230182309832423))
 isc_check_constraints_db1#!#dbo#!#test_udd_c_udd1_check#!#((c_udd1 < '2020-10-20 09:00:00'))
 isc_check_constraints_db1#!#dbo#!#test_udd_c_udd2_check#!#((c_udd2 < '09:00:00'))
-isc_check_constraints_db1#!#dbo#!#test_upper_a_check#!#((sys.upper(a) IN ('A', 'B')))
+isc_check_constraints_db1#!#dbo#!#test_upper_a_check#!#((upper(a) IN ('A', 'B')))
```

Comment:
why this diff?

Reply:
the column `a` of table `test_upper` is of type `sys.BPCHAR`, previously we had a definition of `upper()` function as `sys.UPPER(sys.BPCHAR)` so we were getting `sys.` prepended. But now as we had removed that definition `upper()` is not getting prepended with `sys` here.